src/sentry/models/releasethreshold.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import FlexibleForeignKey, Model, region_silo_only_model
+
+
+@region_silo_only_model
+class ReleaseThreshold(Model):
+    __relocation_scope__ = RelocationScope.Excluded
+
+    class ReleaseThresholdType(models.TextChoices):
```

Comment:
rather than using text here, might it be better to something like this https://github.com/getsentry/sentry/blob/b1582bdd8b0841317271936aeac1918e93ba4b89/src/sentry/constants.py#L462-L478

and use `ReleaseThresholdType`.as_choices()

Reply:
The reason I'm doing it differently is because I'm using built in Django functionality that wasn't available when the referenced code was written (it was added in Django 3 - [source](https://stackoverflow.com/questions/54802616/how-can-one-use-enums-as-a-choice-field-in-a-django-model)). What we did previously was just a makeshift version of what is now directly supported by Django.

Is there some reason why it's better with the `as_choices` option, or something wrong with `TextChoices`?

Reply:
it's actually that i generally see us using some kind of mapping from an integer to an actual value. this _might_ be faster for lookup but i'm not entirely sure